.github/workflows/ci.yaml

```diff
@@ -17,36 +17,25 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - '16.x'
-          - '18.x'
-          - '19.x'
-          - '20.x'
+          - current
+          - lts/*
+          - lts/-1
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Setup Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-
-      - name: Cache dependencies
-        uses: actions/cache@v2
-        with:
-          path: ~/.npm
-          key: npm-${{ matrix.node-version }}-${{ hashFiles('package*.json') }}
-          restore-keys: |
-            npm-${{ matrix.node-version }}-
+          cache: npm
 
       - name: Install dependencies
-        run: |
-          npm install --ignore-scripts
+        run: npm ci
 
       - name: Build
-        run: |
-          npm run build --if-present
+        run: npm run build --if-present
```

Comment:
`run: |` notation is better because it lets you add more lines to the command without having to change existing lines.